src/main/java/DatabaseConfig.java

```diff
@@ -0,0 +1,28 @@
+
+import org.postgresql.ds.PGSimpleDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+@PropertySource(value = {"classpath:db.properties"})
+public class DatabaseConfig {
+    private static final String DB_URL = "db.url";
+    private static final String DB_USERNAME = "db.username";
+    private static final String DB_PASSWORD = "db.password";
+
+    @Autowired
+    private Environment environment;
+
+    @Bean
+    public JdbcTemplate jdbcTemplate() {
+        PGSimpleDataSource dataSource = new PGSimpleDataSource();
+        dataSource.setUrl(environment.getProperty(DB_URL));
+        dataSource.setUser(environment.getProperty(DB_USERNAME));
+        dataSource.setPassword(environment.getProperty(DB_PASSWORD));
```

Comment:
You can do so, technically it'll work but would be better to specify 2 beans - data source and JDBC template. So, create a bean from the data source.